### PR TITLE
Adding a simplified pull request template

### PR DESCRIPTION
**Pull request by patkar**
| Q | A
| --- | ---
| Fixed tickets | none
| License | MIT

#### Description

A pull request template to have at minimum a description and the fixed tickets for documentation pull requests.

#### Purpose

IMHO a documentation PR doesn't need as much information as sulu-io/sulu#2080 adds.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+| Q | A
+| --- | ---
+| Fixed tickets | [comma separated list of tickets fixed by the PR -> `fixes #num`]
+| License | MIT
+
+#### Description
```
Comment (webmozart) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd change this to "What did you change?"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Add as much details as possible about your changes. The pull request description
+helps the documentation review and it serves as a reference when the
+documentation change is merged.
```
Comment (webmozart) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd simplify this to:

> Use simple words to describe what this PR does.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+#### Purpose
```
Comment (webmozart) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd change this to "Why?"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+If your pull request is about adding a new documentation or modifying an
+existing one, explain the rationale for the changes.
```
Comment (webmozart) [inline .github/PULL_REQUEST_TEMPLATE.md]: Again, I'd simplify to:

> Why should we merge your PR?

**Comment by patkar** [comment]
Modified it to be like @webmozart PR template from sulu-io/sulu#2085, because that's really all, which is needed (:

**Comment by patkar** [comment]
Any open pain points @webmozart @danrot @wachterjohannes ?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+| Q | A
+| --- | ---
+| Fixed tickets | fixes #issuenum
+| License | MIT
+
+#### What's in this PR?
+
+Explain the contents of the documentation change.
```
Comment (webmozart) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would use the same wording for this and the next section as in https://github.com/sulu-io/sulu/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

Comment (patkar) [inline .github/PULL_REQUEST_TEMPLATE.md]: 1:1?

Comment (webmozart) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think it makes sense no? "Which topic does the PR cover?" is ambiguous and overlaps with the first section. "Which problem does the PR fix?", OTOH, clearly asks for reasons why a PR is needed.

**Comment by patkar** [comment]
Updated.

**Comment by webmozart** [comment]
Thanks a lot @patkar! :)

ping @sulu-io/core-developer

**Comment by wachterjohannes** [comment]
@webmozart i would say merge it

**Comment by danrot** [comment]
Fine for me too.

**Pull request merged**